cinch/playbooks/jenkins_backup.yml

```diff
@@ -0,0 +1,97 @@
+# This playbook is specifically provided for scenarios where cinch will be used
+# to manage configurations of Jenkins masters that were *not* created by cinch,
+# but by some other method, either manually or via another configuration
+# management tool.  In this case, it's highly recommended to configure backups
+# before running cinch against a Jenkins master for the first time.
+
+# For this playbook to succeed, the thinBackup plugin must be installed, a
+# valid Jenkins user with admin rights must be provided, along with a target
+# directory for the backup data to live.
+
+# It's important to avoid submitting a Jenkins admin username that does not
+# exist on the target Jenkins master.  If this happens, then the
+# jenkins_user_api module will happily create a useless Jenkins user with the
+# name given, which is best avoided.
+
+# Example:
+# ansible-playbook -u root --private-key /path/to/key
+# -i 'jenkins.example.com', jenkins_backup.yml
+# -e "jenkins_admin=example_username"
+# -e "backup_dir=/var/lib/jenkins/thinBackup_data"
+
+- hosts: all
+  vars:
+    jenkins_url: "http://localhost:8080"
+    jenkins_user: jenkins
+    jenkins_backup:
+      directory: "{{ backup_dir }}"
+      full_schedule: H 0 * * 0
+      diff_schedule: H 0 * * *
+      max_sets: 2
+      exclude: ''
+      wait_for_idle: true
+      quiet_mode_timeout: 480
+      build_results: false
```

Comment:
I think these are sane defaults, and that those users can check the boxes if they wish. We should create documentation somewhere (if we don't already have it) about how to do that via ansible vars.